app/lib/actions/katello/host/remove_subscriptions.rb

```diff
@@ -0,0 +1,29 @@
+module Actions
+  module Katello
+    module Host
+      class RemoveSubscriptions < Actions::Base
+        middleware.use Actions::Middleware::KeepCurrentUser
+
+        def plan(host, entitlements)
+          entitlements.each do |entitlement|
+            plan_action(::Actions::Candlepin::Consumer::RemoveSubscription, :uuid => host.subscription_facet.uuid,
+                        :entitlement_id => entitlement['id'])
+            plan_self(:host_name => host.name)
+          end
```

Comment:
that's actually the purpose of the Dynflow::Action::Rescue::Skip  below.  Bulk makes more sense where there may a very large number of items and planning may take too long, or you want to make it easier to determine why 3 out of the 500 failed.   I don't think it makes sense to use here, but good question!